lua/MenuTweaks.lua

```diff
@@ -64,15 +68,19 @@ if string.lower(RequiredScript) == "lib/managers/menumanager" then
 		end
 	end
 
-	local open_contract_node_orig = MenuCallbackHandler.open_contract_node
-	function MenuCallbackHandler:open_contract_node(...)	-- Contract Broker
-		load_lobby_settings()
-		return open_contract_node_orig(self, ...)
-	end
```

Comment:
i deleted this, because MenuCrimeNetContractInitiator:modify_node is triggered in both cases anyways.